From 7c080de2d003b433305224374c9c2c0b65175046 Mon Sep 17 00:00:00 2001
From: "Wong, Vincent Por Yin" <vincent.por.yin.wong@intel.com>
Date: Fri, 3 Apr 2020 14:56:01 +0800
Subject: [PATCH 3/3] makefile: remove check for reallocarray()

Libbpf has a script that checks the host system for a compatible
reallocarray() implementation. This check is inaccurate in yocto
BSP builds and is removed to resolve compilation errors.

Signed-off-by: Wong, Vincent Por Yin <vincent.por.yin.wong@intel.com>
---
 src/Makefile | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 17c04b5..7dc3bfa 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -10,11 +10,6 @@ TOPDIR = ..
 INCLUDES := -I. -I$(TOPDIR)/include -I$(TOPDIR)/include/uapi
 ALL_CFLAGS := $(INCLUDES)
 
-FEATURE_REALLOCARRAY := $(shell $(TOPDIR)/scripts/check-reallocarray.sh)
-ifneq ($(FEATURE_REALLOCARRAY),)
-	ALL_CFLAGS += -DCOMPAT_NEED_REALLOCARRAY
-endif
-
 SHARED_CFLAGS += -fPIC -fvisibility=hidden -DSHARED
 
 CFLAGS ?= -g -O2 -Werror -Wall
-- 
2.17.0

